README.md

```diff
@@ -144,6 +144,75 @@ This section applies to Solidity and JavaScript projects alike.
+
+Ideally, a PR is small, limited to a single objective, and in response to a specific GitHub issue.
+
+The PR is the only place any team member is expected to consult for information about the PR or its lifecycle.
+
+### Summary
+
+* a PR is its source of truth
+* a PR should be connected to the relevant issue(s)
+* the reviewer who takes up review formally as an item of work should self-assign on the PR
+* when the review ends, because the assignee either completes the review or will not complete the review, assignment should be cleared
+* comments on the PR should be limited to the PR
+* comments about the PR should only be made on the PR
+
+### Format
+
+The PR should be connected to any issues to which it is a response. If the PR resolves an issue, that should be indicated in the body of the PR (e.g., "Resolves #70", "Fixes #71", etc.).
+
+The PR's title and description should collectively address the following:
+* what is the purpose of these changes
+* what has changed
+* any information pertinent to this particular PR's review that are not generally true of PRs in the repo (e.g., assumptions, dependencies, departures from normal practice, offline agreements, etc.)
+* any additional information helpful for review (and posterity!)
+
+### Review
+
+If a PR is ready for review, the PR should be moved into the review pipeline (if it was not already there) and any assignees should be cleared.
+
+#### Assignment
+
+_The following assumes there is a single assignee for a PR. Should there be multiple assignees, apply sensible adjustments._
+
+The question of the best use of "Reviewers" and "Assignees" on GitHub appears to be open to debate. However, two features of our setup resolve the matter:
+* GitHub automatically lists anyone who adds a comment to a PR as a reviewer
+* ZenHub does not filter by reviewers, only by assignees and authors
+
+**On a PR, the person formally taking up the task of review, and who therefore accepts responsibility if she decides the work can be merged, should be set as the assignee.** This supports quick and simple assessment of resources via the ZenHub board. Others can, of course, review if they have capacity/interest and add their comments (doing so will automatically list them as reviewers), but they are not considered responsible for completion of the review.
+
+Generally speaking, the assignee self-assigns to the PR and, ideally, self-clears as necessary. If the assignee cannot complete review, that assignment should be cleared from the PR. In doing so, the PR will appear unassigned in the review pipeline on the ZenHub board and others will know that PR's review is available as a task to pick up. If the PR is not in a reviewable state, this can be reflected by moving the PR out of the review pipeline and/or assignment to the person responsible for addressing the state of the PR, according to any workflow policies or as deemed sensible.
```

Comment:
The current process, I think, is to move the PR from "Review" to "Bounced" and unassign. The original author can pick it up from there.

Reply:
@schemar: ah, yes—will update!

Reply:
I can't seem to find the update here 🧐